tests/test_aims_ui.py

```diff
@@ -2,27 +2,50 @@
 import tempfile
 import json
 import pytest
+from flask import url_for
 
 from src import aims_ui
 
+
 def test_info_page(client):
-    """Test the info page exists, and the ENV variable is being displayed."""
-    required_keys = ['ENV','PLATFORM','VERSION']
+  """Test the info page exists, and the ENV variable is being displayed."""
+  required_keys = ['ENV', 'PLATFORM', 'VERSION']
+
+  rv = client.get('/info')
+  response_dict = json.loads(rv.data)
+
+  assert all(k in response_dict for k in required_keys)
+
+
+def test_static_pages_are_200(client):
+  """Basic Check to see if pages are present, returning a 200, and also contain some form of content"""
+  urls = [
+      'uprn',
+      'landing',
+      'postcode',
+      'singlesearch',
+      'typeahead',
+      'multiple_address',
+  ]
+
+  for url in urls:
+    response = client.get('/' + url)
+    if response.status_code == 404:
+      raise Exception(
+          f'Page not found error for {url}, got response: {response}')
 
-    rv = client.get('/info')
-    response_dict = json.loads(rv.data)
+    assert response.status_code < 400
```

Comment:
These checks don't match the docstring above. It isn't checking for a 200, it's checking for anything under a 400, and will end up matching status codes you don't want.

This should either directly check for `== 200`, or `in [200, 201]' if there are multiple valid return codes that you're anticipating.

Reply:
Fixed this issueby changing it to `== 200`